bin/portageq

```diff
@@ -1451,9 +1451,11 @@ try:
 
     def main(argv):
         argv = portage._decode_argv(argv)
-
-        nocolor = os.environ.get("NOCOLOR")
-        if nocolor in ("yes", "true"):
+        if os.environ.get("NO_COLOR"):
+            nocolor = os.environ.get("NO_COLOR") != ""
```

Comment:
I think comparison with `""` is not correct, because `NO_COLOR` could be unset. Both `None` and `""` are logically false though:

```suggestion
            nocolor = os.environ.get("NO_COLOR")
```

(Same issue in more places below.)

Reply:
Oh sorry it should have been == right not !=

Reply:
It should have been == not != right ? I'm sorry I'll rectify it

Reply:
Wait wouldn't nocolor be "" like you mentioned above and it won't be a boolean right ? Is that an expected behaviour ?

Reply:
But in the condition if it's not unset it can be an empty str or srt of len > 0

Reply:
Should I add a condition to deal with None too ?

Reply:
> Should I add a condition to deal with None too ?

I think you can check for both `None` and `""` at once, because in Python a string has boolean value true if (and only if) it exists and is non-empty. 

See here for the test in pkgcheck:
https://gitweb.gentoo.org/proj/pkgcore/pkgcheck.git/tree/src/pkgcheck/scripts/pkgcheck_scan.py#n404

Reply:
Yes I understand